Harvest.Web/ClientApp/src/Quotes/ProjectDetail.tsx

```diff
@@ -120,46 +120,20 @@ export const ProjectDetail = (props: Props) => {
                 }
               />
             </Col>
-            <Col md="4">
-              <Label for="rate">Rate</Label>
-              <InputGroup>
-                <InputGroupAddon addonType="prepend">
-                  <InputGroupText>$</InputGroupText>
-                </InputGroupAddon>
-                <Input
-                  type="number"
-                  id="rate"
-                  value={props.quote.acreageRate}
-                  disabled
-                />
-              </InputGroup>
+            <Col md="2">
+              <Label for="rate">Rate</Label> <br />${props.quote.acreageRate}
```

Comment:
let's throw a `formatCurrency()` in here so it looks nice, like we do with total down below

Reply:
So like this? 
<Label for="rate">Rate</Label> <br />${formatCurrency(props.quote.acreageRate)}

Reply:
`<Label for="rate">Rate</Label> <br />${formatCurrency(props.quote.acreageRate)}`

Reply:
yes perfect